### PR TITLE
KVStore: More log for Parallel Prehandle

### PR DESCRIPTION
**Pull request by CalvinNeo**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tiflash/issues/8081

Problem Summary:

### What is changed and how it works?

```commit-message

```

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Review by Lloyd-Pottiger — APPROVED** [review]

File: dbms/src/Storages/KVStore/FFI/SSTReader.cpp [hunk 1]
```diff
@@ -129,7 +131,8 @@ MonoSSTReader::MonoSSTReader(
             "Seek cf {} to {}, split_id={}",
```
Comment (JaySon-Huang) [inline dbms/src/Storages/KVStore/FFI/SSTReader.cpp:131]:
```suggestion
            "Seek cf {} to {}, split_id={} region_id={}",
```

File: dbms/src/Storages/DeltaMerge/Decode/SSTFilesToBlockInputStream.cpp [hunk 2]
```diff
@@ -460,17 +465,24 @@ bool SSTFilesToBlockInputStream::maybeSkipBySoftLimit(ColumnFamilyType cf, SSTRe
...
+                tikv_key.toDebugString(),
                 magic_enum::enum_name(cf),
                 soft_limit.value().split_id,
                 region->id());
             LOG_INFO(
                 log,
-                "Re-Seek after start_raw={} start_pk={} to {}, current_pk={} cf={} split_id={} region_id={}",
+                "Re-Seek after skipped_times={} start_raw={} start_pk={} current_raw={}, current_pk={}, cf={}, "
+                "split_id={}, "
+                "region_id={}",
```
Comment (JaySon-Huang) [inline dbms/src/Storages/DeltaMerge/Decode/SSTFilesToBlockInputStream.cpp:484]:
```suggestion
                "Re-Seek after skipped_times={} start_raw={} start_pk={} current_raw={} current_pk={} cf={} "
                "split_id={} region_id={}",
```

File: dbms/src/Storages/DeltaMerge/Decode/SSTFilesToBlockInputStream.cpp [hunk 3]
```diff
@@ -460,17 +465,24 @@ bool SSTFilesToBlockInputStream::maybeSkipBySoftLimit(ColumnFamilyType cf, SSTRe
         {
             RUNTIME_CHECK_MSG(
                 current_truncated_ts > start_limit,
-                "current pk decreases as reader advances, start_raw={} start_pk={} current={} cf={} split_id={} "
+                "current pk decreases as reader advances, skipped_times={} start_raw={} start_pk={} current_pk={} "
+                "current_raw={}, "
+                "cf={}, split_id={}, "
```
Comment (JaySon-Huang) [inline dbms/src/Storages/DeltaMerge/Decode/SSTFilesToBlockInputStream.cpp:470]:
```suggestion
                "current pk decreases as reader advances, skipped_times={} start_raw={} start_pk={} current_pk={} "
                "current_raw={} cf={} split_id={}, "
```

File: dbms/src/Storages/KVStore/MultiRaft/PrehandleSnapshot.cpp [hunk 4]
```diff
@@ -207,7 +207,7 @@ static inline std::tuple<ReadFromStreamResult, PrehandleResult> executeTransform
             res = ReadFromStreamResult{.error = abort_reason.value(), .extra_msg = "", .region = new_region};
         }
         auto keys_per_second = (sst_stream->getProcessKeys().write_cf + sst_stream->getProcessKeys().lock_cf
-                                + sst_stream->getProcessKeys().write_cf)
+                                + sst_stream->getProcessKeys().default_cf)
```
Comment (JaySon-Huang) [inline dbms/src/Storages/KVStore/MultiRaft/PrehandleSnapshot.cpp:210]: 😂 seems we should backport this to the affected release-x branch

Comment (CalvinNeo) [reply dbms/src/Storages/KVStore/MultiRaft/PrehandleSnapshot.cpp:210]: It's merged into master last week.

**Review by JaySon-Huang — APPROVED** [review]
lgtm

File: dbms/src/Storages/KVStore/MultiRaft/PrehandleSnapshot.cpp (hunk 4 above)
Comment (JaySon-Huang) [reply dbms/src/Storages/KVStore/MultiRaft/PrehandleSnapshot.cpp:210]: > It's merged into master last week.

ref https://github.com/pingcap/tiflash/pull/9131

**Pull request merged**